src/Mvc/Mvc.Core/src/JsonResult.cs

```diff
@@ -80,5 +83,15 @@ public override Task ExecuteResultAsync(ActionContext context)
             var executor = services.GetRequiredService<IActionResultExecutor<JsonResult>>();
             return executor.ExecuteAsync(context, this);
         }
+
+        /// <summary>
+        /// Write the result as JSON to the HTTP response.
+        /// </summary>
+        /// <param name="httpContext">The <see cref="HttpContext"/> for the current request.</param>
+        /// <returns>A task that represents the asynchronous execute operation.</returns>
+        public ValueTask WriteResponseAsync(HttpContext httpContext)
```

Comment:
Implicit implementation right?

Reply:
I'm not sure what you're asking here. Does this mean that you want `JsonResults`'s `IResult` implementation to be the same as the `ActionResult` implementation and use the `IActionResultExecutor<JsonResult>` service?

Reply:
```suggestion
        ValueTask IResult.WriteResponseAsync(HttpContext httpContext)
```

Reply:
D'oh! 😆